authentication/index.adoc

```diff
@@ -0,0 +1,57 @@
++
+[NOTE]
+====
+Users must specify either a default or custom OAuth client that receives and uses the token in their requests for the OAuth token.
+====
+
+* xref:../authentication/managing-oauth-access-tokens.adoc#managing-oauth-access-tokens[Viewing and managing user-owned OAuth access tokens].
+* Managing cloud provider credentials using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[Cloud Credentials Operator]: Cluster components use cloud provider credentials to get permissions required to perform cluster-related tasks.
+* Impersonating a system admin user: You can grant cluster administrator permissions to a user by xref:../authentication/impersonating-system-admin.adoc#impersonating-system-admin[impersonating a system admin user].
+
+[id="authorization-overview"]
+== About authorization in {product-title}
+Authorization involves determining whether the identified user has permissions to perform the requested action.
+
+Administrators can define permissions and assign them to users using the xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[RBAC objects, such as rules, roles, and bindings]. To understand how authorization works in {product-title}, see xref:../authentication/using-rbac.adoc#evaluating-authorization_using-rbac[Evaluating authorization].
+
+You can also control access to an {product-title} cluster through xref:../authentication/using-rbac.adoc#rbac-projects-namespaces_using-rbac[projects and namespaces].
+
+Along with controlling user access to a cluster, you can also control the actions a pod can perform and the resources it can access using xref:../authentication/managing-security-context-constraints.adoc#managing-pod-security-policies[security context constraints (SCCs)].
+
+You can manage authorization for {product-title} through the following tasks:
+
+* Viewing xref:../authentication/using-rbac.adoc#viewing-local-roles_using-rbac[local] and xref:../authentication/using-rbac.adoc#viewing-cluster-roles_using-rbac[cluster] roles and bindings.
+* Creating a xref:../authentication/using-rbac.adoc#creating-local-role_using-rbac[local role] and assigning it to a user or group.
+* Creating a cluster role and assigning it to a user or group: {product-title} includes a set of xref:../authentication/using-rbac.adoc#default-roles_using-rbac[default cluster roles]. You can create additional xref:../authentication/using-rbac.adoc#creating-cluster-role_using-rbac[cluster roles] and xref:../authentication/using-rbac.adoc#adding-roles_using-rbac[add them to a user or group].
+* Creating a cluster-admin user: By default, your cluster has only one cluster administrator called `kubeadmin`. You can xref:../authentication/using-rbac.adoc#creating-cluster-admin_using-rbac[create another cluster administrator]. Before creating a cluster administrator, ensure that you have configured an identity provider.
```

Comment:
There's also the `system:admin` which is the actual real administrator, `kubeadmin` won't work in many scenarios where certain parts of the cluster get broken

The difference between the two is that `kubeadmin` is a more of an OpenShift-kind of a user - you can log in as this user with a username and password. `system:admin` is a kubernetes user that only uses client-certificate authentication. I'm not sure how to get all of this information in here, though.

Reply:
@stlaz https://docs.openshift.com/container-platform/4.9/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-identity-provider also mentions just the kubeadmin user. Should we modify the content everywhere?